src/open_inwoner/accounts/views/cases.py

```diff
@@ -386,7 +386,18 @@ def get_upload_info_context(self, case: Zaak):
         else:
             external_upload_url = ""
 
+        try:
+            case_type_config = ZaakTypeConfig.objects.get(
+                identificatie=case.zaaktype.identificatie,
+                catalogus__url=case.zaaktype.catalogus,
+            )
```

Comment:
This should have used the ZaakTypeConfig queryset methods. (`filter_catalog()`, possibly also `filter_case_type()`)

But this function seems to do a lot of repeated queries that could be simplified (I see `get_visible_zt_configs_for_case_type_identification()` twice, and this new addition overlaps with it as well.

Also I see the old comments in the queryset how these should be refactored, and I'm not sure if you can safely do a `.get()` (and there isn't a case that returns multiple objects) maybe I should pick this up and do the refactoring and amend the PR.

Reply:
Yes, it would be nice. Thank you Bart